app/views/api/v1/shelters/outdated.json.jbuilder

```diff
@@ -0,0 +1,13 @@
+json.cache! @outdated do
```

Comment:
I don't see any configuration for a cache expiration time.  Unless I have missed it (very possible) `json.cache!` will cache indefinitely.  Adding an `expires_in` parameter is probably smart.  Also, how do we invalidate the cache if an update is made before expiration?

Reply:
Also curious about why you didn't keep `@shelters` as the variable.  Would have allowed you to keep the same variable for apply_params function instead of replaces the instance variables with local variables.

Reply:
Started out trying to be explicit, and things got deeper and deeper. I may switch this back and clean that up. I do like the refactoring of `apply_params`, so I'll likely keep that anyway.

Regarding cache, I'm just mimicking what's there already. I agree we should probably look at that, but I think that's a new issue unrelated to this, and should have it's own PR.